### PR TITLE
add choose file button, fix styling

### PR DESCRIPTION
**Pull request by Nanobot567**
Co-authored by: @saturnaliam :3rd_place_medal:

### CODE REVIEW

File: convert_to_js.py [hunk 1]
```diff
@@ -22,6 +22,8 @@
 
         data.append(point)
 
+# print(json.dumps(data))
```
Comment (nkalupahana) [inline convert_to_js.py]: Remove

File: js/funcs.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+const degreesToRadians = (x) => {
+  return (x * Math.PI) / 180;
+};
+
+function clamp(num, min, max) {
+  return Math.min(Math.max(num, min), max);
+};
+
+function csvJSON(csv) {
```
Comment (nkalupahana) [inline js/funcs.js:11]: Use `let` and `const`, not `var`

File: index.html [hunk 3]
```diff
@@ -1,183 +1,242 @@
+<!DOCTYPE html>
 <html>
+
 <head>
-    <link href="style.css" rel="stylesheet" type="text/css" />
+  <link href="style.css" rel="stylesheet" type="text/css" />
+  <title>Simple Scope</title>
```
Comment (nkalupahana) [inline index.html]:
```suggestion
  <title>SimpleScope</title>
```

File: style.css [hunk 4]
```diff
@@ -1,5 +1,10 @@
+html {
```
Comment (nkalupahana) [inline style.css:1]: Make 2/4 spaces consistent throughout

**Review by nkalupahana — CHANGES_REQUESTED** [review]

File: convert_to_js.py (hunk 1 above)
Comment (Nanobot567) [reply convert_to_js.py]: convert_to_js was removed.

File: js/funcs.js (hunk 2 above)
Comment (Nanobot567) [reply js/funcs.js:11]: fixed in 3191ba1.

File: index.html (hunk 3 above)
Comment (Nanobot567) [reply index.html]: fixed in 3191ba1.

File: style.css (hunk 4 above)
Comment (Nanobot567) [reply style.css:1]: fixed in 3191ba1.

**Review by nkalupahana — APPROVED** [review]

**Pull request merged**